### PR TITLE
Change int8v2 CAPI unit test name and add log in the prediction stage

### PR DESCRIPTION
**Pull request by zhaify**
fix two remaining issues of PR [18111](https://github.com/PaddlePaddle/Paddle/pull/18111#issuecomment-502527324)
* add log refer [comment](https://github.com/PaddlePaddle/Paddle/pull/18111#issuecomment-502527324)
* fix unit test name issue refer [comment](https://github.com/PaddlePaddle/Paddle/pull/18111#issuecomment-502602193)

### CODE REVIEW

File: paddle/fluid/inference/tests/api/tester_helper.h [hunk 1]
```diff
@@ -335,24 +335,26 @@ void PredictionRun(PaddlePredictor *predictor,
...
         predictor->ZeroCopyRun();
       }
-      elapsed_time += run_timer.toc();
+    }
+    int predicted_num = (i + 1) * FLAGS_batch_size;
+    if (predicted_num % 100 == 0) {
+      LOG(INFO) << predicted_num << " samples";
     }
   }
+  elapsed_time += run_timer.toc();
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/tester_helper.h]: 这里改的不对，这样会把`ConvertPaddleTensorToZeroCopyTensor`，以及打印LOG的时间都算进去了。应该只算`predictor->ZeroCopyRun();`或者`predictor->Run();`的时间。

Comment (luotao1) [reply paddle/fluid/inference/tests/api/tester_helper.h]:
```
diff --git a/paddle/fluid/inference/tests/api/tester_helper.h b/paddle/fluid/inference/tests/api/tester_helper.h
index eb786196a..3eba802d9 100644
--- a/paddle/fluid/inference/tests/api/tester_helper.h
+++ b/paddle/fluid/inference/tests/api/tester_helper.h
@@ -337,13 +337,14 @@ void PredictionRun(PaddlePredictor *predictor,
   ProfilerStart("paddle_inference.prof");
 #endif
   if (!FLAGS_zero_copy) {
-    run_timer.tic();
     for (int i = 0; i < iterations; i++) {
+      run_timer.tic();
       for (int j = 0; j < num_times; j++) {
         predictor->Run(inputs[i], &(*outputs)[i], FLAGS_batch_size);
       }
+      elapsed_time += run_timer.toc();
+      if (i % 100 == 0) LOG(INFO) << i << " samples";
     }
-    elapsed_time = run_timer.toc();
   } else {
     for (int i = 0; i < iterations; i++) {
       ConvertPaddleTensorToZeroCopyTensor(predictor, inputs[i]);
```
这是我改的，打log部分可以增强下。

Comment (zhaify) [reply paddle/fluid/inference/tests/api/tester_helper.h]: done with remove time count on `ConvertPaddleTensorToZeroCopyTensor`

File: paddle/fluid/inference/tests/api/tester_helper.h [hunk 2]
```diff
@@ -335,24 +335,28 @@ void PredictionRun(PaddlePredictor *predictor,
 #ifdef WITH_GPERFTOOLS
   ProfilerStart("paddle_inference.prof");
 #endif
-  if (!FLAGS_zero_copy) {
-    run_timer.tic();
-    for (int i = 0; i < iterations; i++) {
+
+  for (int i = 0; i < iterations; i++) {
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/tester_helper.h]: - for循环移到分支外，整体测试时间会拉长吧？
- 建议把for循环移到分支里面，每个分支打印一下log。

**Comment by luotao1** [comment]
PR标题请增强下。

File: paddle/fluid/inference/tests/api/tester_helper.h (hunk 2 above)
Comment (zhaify) [reply paddle/fluid/inference/tests/api/tester_helper.h]: done

**Review by luotao1 — APPROVED** [review]
LGTM

**Review by wojtuss — APPROVED** [review]

**Comment by luotao1** [comment]
please cherry-pick this PR and #17909 to the release/1.5 branch.

**Pull request merged**